definitions/output/all/requests_stable.js

```diff
@@ -0,0 +1,176 @@
+let month = '2024-08-01',
+    month_YYYYMM = constants.fn_past_month(month).replace('-', '').substring(0, 6);
+
+publish("requests_stable", {
+    type: "incremental",
+    schema: "all",
+    bigquery: {
+        partitionBy: "date",
+        clusterBy: ["client", "is_root_page", "type", "rank"],
+        requirePartitionFilter: true,
+    },
+    columns: {
+      date: "YYYY-MM-DD format of the HTTP Archive monthly crawl",
+      client: "Test environment: desktop or mobile",
+      page: "The URL of the page being tested",
+      is_root_page: "Whether the page is the root of the origin.",
+      root_page: "The URL of the root page being tested",
+      rank: "Site popularity rank, from CrUX",
+      url: "The URL of the request",
+      is_main_document: "Whether this request corresponds with the main HTML document of the page, which is the first HTML request after redirects",
+      type: "Simplified description of the type of resource (script, html, css, text, other, etc)",
+      index: "The sequential 0-based index of the request",
+      payload: "JSON-encoded WebPageTest result data for this request",
+      summary: "JSON-encoded summarization of request data",
+      request_headers: {
+        description: "Request headers",
+        columns: {
+          name: "Request header name",
+          value: "Request header value",
+        },
+      },
+      response_headers: {
+        description: "Response headers",
+        columns: {
+          name: "Response header name",
+          value: "Response header value",
+        },
+      },
+      response_body: "Text-based response body",
```

Comment:
```suggestion
```

@tunetheweb @rviscomi 
I saw couple of mentions that nobody really uses this column.
Did you discuss if we need to keep it for any future use-cases?

I would recommend to update custom_metrics and wait for a next crawl data instead.

Reply:
There are definitely some use cases where it's necessary to query the raw response bodies. IMO we should keep it to support those use cases, and it shouldn't affect the time/cost for any queries that don't read it directly.

Reply:
Agreed. Keep it, but massively discourage its use unless a very convincing use case is what I’ve been doing.